subsys/bluetooth/controller/ll_sw/ull.c

```diff
@@ -646,9 +664,9 @@ void ll_rx_dequeue(void)
 			 * enabled status bitmask
 			 */
 			bm = (IS_ENABLED(CONFIG_BT_OBSERVER) &&
-			      ull_scan_is_enabled(0) << 1) |
-			     (IS_ENABLED(CONFIG_BT_BROADCASTER) &&
-			      ull_adv_is_enabled(0));
+				ull_scan_is_enabled(0) << 1) |
```

Comment:
revert indentation changes.

Reply:
(See previous comment.)

Reply:
did you miss this one? doesn't seem to have been reverted